### PR TITLE
Added remote git functionality

### PR DESCRIPTION
**Pull request by jlehtimaki**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 release
 coverage.out
 vendor
+.idea
```
Comment (tboerger) [inline .gitignore]: We are not adding any IDE-specific ignores, please add it to your local global gitignore.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM golang:1.13 AS builder
```
Comment (tboerger) [inline Dockerfile]: I don't get this new Dockerfile, why?

File: cmd/drone-docker/main.go [hunk 3]
```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 
-	"github.com/drone-plugins/drone-docker"
+	"github.com/jlehtimaki/drone-docker"
```
Comment (tboerger) [inline cmd/drone-docker/main.go]: Why are you changing the import path?

File: cmd/drone-docker/main.go [hunk 4]
```diff
@@ -121,11 +121,10 @@ func main() {
 			EnvVar: "PLUGIN_CONTEXT",
 		},
 		cli.StringSliceFlag{
-			Name:     "tags",
-			Usage:    "build tags",
-			Value:    &cli.StringSlice{"latest"},
-			EnvVar:   "PLUGIN_TAG,PLUGIN_TAGS",
-			FilePath: ".tags",
```
Comment (tboerger) [inline cmd/drone-docker/main.go]: No, this got to stay.

File: docker/ecr/Dockerfile.linux.amd64 [hunk 5]
```diff
@@ -1,4 +1,4 @@
-FROM plugins/docker:linux-amd64
+FROM lehtux/drone-docker
```
Comment (tboerger) [inline docker/ecr/Dockerfile.linux.amd64]: This will also never get into this repository.

File: go.mod [hunk 6]
```diff
@@ -1,4 +1,4 @@
-module github.com/drone-plugins/drone-docker
+module github.com/jlehtimaki/drone-docker
```
Comment (tboerger) [inline go.mod]: Same as above

File: docker.go [hunk 7]
```diff
@@ -156,6 +167,10 @@ func commandLogin(login Login) *exec.Cmd {
 	)
 }
 
+func commandGit(build Build) *exec.Cmd {
+	return exec.Command("git","clone", build.GitUrl)
```
Comment (tboerger) [inline docker.go]: I think git is not available within the image.

**Comment by jlehtimaki** [comment]
Sorry this was an accidental PR.

**Pull request closed without merging**